apps/expo/src/app/privacy-policy/index.tsx

```diff
@@ -0,0 +1,225 @@
+import React, { useState } from 'react'
+import { Switch, Text, View } from 'react-native'
+import { ScrollView } from 'react-native-gesture-handler'
+import { SafeAreaView } from 'react-native-safe-area-context'
+
+import Label from '~/components/ui/label/Label'
+
+const tailwindClasses = {
+  mainTile: 'm-4 p-4 rounded-lg bg-bb-dark-gray',
+  mainTileTitle: 'text-2xl text-slate-200',
+  mainTileItem: 'flex-row',
+  navigationListItemLabel: 'flex-auto m-2 self-center text-slate-200',
+  navigationListItemChevron: 'self-center text-slate-200',
+}
+
+const PrivacyPolicy = () => {
+  const [isEnabled, setIsEnabled] = useState(false)
+  const toggleSwitch = () => setIsEnabled((previousState) => !previousState)
+
+  const privacyToggleSwitch = async () => {}
+  // The privacy policy was AI generated via a prompt to chatGPT, the link to the prompt is below
+  // https://chat.openai.com/share/edbfaba8-f225-4168-8d86-c8263ed9ddc5
+  return (
+    <SafeAreaView
+      className='bg-bb-slate-100 flex-1'
+      style={{ backgroundColor: '#1e1e1e', flex: 1 }}
+    >
+      <ScrollView>
+        <View>
+          <Text
+            className='text-2xl text-slate-200'
+            style={{
+              fontSize: 40,
+              textAlign: 'center',
+              paddingHorizontal: 10,
+              fontWeight: 'bold',
+            }}
+          >
+            Privacy Policy
+          </Text>
+
+          <View style={{ paddingHorizontal: 30, flexDirection: 'row' }}>
+            <Label text='Legal Document' textColor='#CFCFCF' backgroundColor='#48476D' />
+            <Text> </Text>
+            <Label text='Human-Friendly' textColor='#CFCFCF' backgroundColor='#48476D' />
+          </View>
+
+          <Text
+            className='text-2xl text-slate-200'
+            style={{ fontSize: 15, textAlign: 'center', paddingHorizontal: 10 }}
+          >
+            {'\n'}Effective Date: [4/19/2024]{'\n'}
+          </Text>
+          <View className={tailwindClasses.mainTile}>
+            <Text className='text-2xl text-slate-200' style={{ paddingHorizontal: 10 }}>
+              Welcome to Barbell, your ultimate companion for fitness and workout routines. At
+              Barbell, we are committed to protecting your privacy and ensuring the security of your
+              personal information. This Privacy Policy outlines how we collect, use, disclose, and
+              protect your information when you use our mobile application ("App") and services.
+              {'\n\n'}
```

Comment:
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/fc206654-b20d-47ce-9809-785a2a147006)

remove the \n\n at the end here, too many spaces comparatively

![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/c30d2f78-2d04-453e-9118-9852a922ce6d)

Reply:
Done, good style correction.